tck/src/main/java/org/eclipse/microprofile/openapi/tck/AppTestBase.java

```diff
@@ -111,6 +111,42 @@ public static String dereference(ValidatableResponse vr, String path) {
         }
     }
 
+    public static String dereference(ValidatableResponse vr, String... paths) {
+        ExtractableResponse<Response> response = vr.extract();
+        String context = "";
+        StringBuilder lookup = new StringBuilder();
+        StringBuilder absolutePath = new StringBuilder();
+
+        for (String path : paths) {
+            lookup.setLength(0);
+
+            if (!context.isEmpty()) {
+                lookup.append(context);
+                lookup.append('.');
+            }
+
+            lookup.append(path);
+            lookup.append(".$ref");
+
+            String ref = response.path(lookup.toString());
+
+            if (ref != null) {
+                absolutePath.setLength(0);
+                absolutePath.append(ref.replaceFirst("^#/?", "").replace('/', '.'));
```

Comment:
restassured is using JsonPath for this functionality (versus Json Pointer).

Reply:
Yes, that's what I meant. If you have a `ref` where one of the keys in the path includes a `/`, it'll be replaced with `~1`. When you do the conversion to JsonPath, I think you would need to change those back.

This is likely to happen if you have a reference to an operation, because a Json Pointer to the operation will include one of the path elements which will likely include a `/`.

For example:
```openapi: 3.0.3
info: 
  title: "test"
  version: "1.0"
paths:
  '/foo/bar':
    get:
      description: "test operation"
      responses:
        200:
          description: OK
```
The test operation would be referenced as `#/paths/~1foo~1bar/get`. I think as JsonPath this would need to become `paths./foo/bar.get`.

I think paths are the only keys likely to contain slashes and I think operations are the only thing you can reference that would include a path in the reference string, as long as you're not doing that it should be fine but I do think a mention in the javadoc that it doesn't handle references with escaped characters would be helpful.

Reply:
Makes sense. I've updated the docs to mention that references with escapes are not supported.